framework/src/org/checkerframework/framework/qual/TypeQualifier.java

```diff
@@ -3,11 +3,22 @@
 import java.lang.annotation.*;
 
 /**
- * A meta-annotation indicating that the annotated annotation is a type
- * qualifier.
+ * This meta-annotation is deprecated.
+ * <p>
  *
- * Examples of such qualifiers: {@code @ReadOnly}, {@code @NonNull}
+ * An annotation will no longer use this meta-annotation. To indicate that an
```

Comment:
I would just add a link to the manual section you wrote instead of this paragraph.

@checker_framework.manual #indicating-supported-annotations Indicating supported annotations